### PR TITLE
add review and rating

### PR DESCRIPTION
**Pull request by bachtung144**
## Related Tickets:
- [#40614](https://edu-redmine.sun-asterisk.vn/issues/40614)

## WHAT this PR does?
add review and rating

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by bachtung144** [comment]
ready

**Review by hoangdv99 — APPROVED** [review]

**Comment by bachtung144** [comment]
ready

File: app/Http/Livewire/DetailsComponent.php [hunk 1]
```diff
@@ -29,7 +30,8 @@ public function render()
     {
         $product = Product::where('slug', $this->slug)->first();
         $images = Images::where('product_id', $product->id)->get();
+        $reviews = Review::where('product_id', $product->id)->get();
```
Comment (huyenttm-0901) [inline app/Http/Livewire/DetailsComponent.php:33]:
```suggestion
        $images = $product->images;
        $reviews = $product->reviews;
```
dùng relationship đi em

File: app/Http/Livewire/User/UserReviewComponent.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        $data['product_id'] = $this->orderProductId;
+        $data['user_id'] = Auth::user()->id;
+        Review::create($data);
+
+        session()->flash('message', __('review.success_review'));
+    }
+
+    public function render()
+    {
+        $orderProduct = Product::where('id', $this->orderProductId)->first();
```
Comment (huyenttm-0901) [inline app/Http/Livewire/User/UserReviewComponent.php:43]: dùng findOrFail()

**Comment by bachtung144** [comment]
ready

File: resources/lang/en/review.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+return [
+  'add_review' => 'Add review for',
+  'your_review' => 'Your review',
+  'see_review' => 'See review',
+  'success_review' => 'Your review has been added successfully!',
+  'write_review' => 'Write Review',
+];
```
Comment (manhtd-1083) [inline resources/lang/en/review.php:9]: indent 4 space

File: resources/lang/vi/review.php [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+return [
+  'add_review' => 'Thêm đánh giá cho',
+  'your_review' => 'Đánh giá của bạn',
+  'see_review' => 'Xem đánh giá',
+  'success_review' => 'Đánh giá của bạn đã được thêm thành công!',
+  'write_review' => 'Viết đánh giá',
+];
```
Comment (manhtd-1083) [inline resources/lang/vi/review.php:9]: tương tự

**Comment by bachtung144** [comment]
ready

**Review by manhtd-1083 — APPROVED** [review]

**Pull request merged**